worldedit-bukkit/src/main/java/com/sk89q/worldedit/bukkit/BukkitServerInterface.java

```diff
@@ -218,6 +220,18 @@ public String getPlatformVersion() {
         return capabilities;
     }
 
+    private static final Collection<SideEffect> SUPPORTED_SIDE_EFFECTS = ImmutableSet.copyOf(EnumSet.of(
```

Comment:
You can use `Sets.immutableEnumSet` instead of `ImmutableSet.copyOf(EnumSet.of` - applies to all cases throughout this PR.